GDSCommon-Demo/GDSCommon-Demo/Mocks/Screens.swift

```diff
@@ -24,15 +25,17 @@ enum Screens: CaseIterable {
             return "GDS Instructions View (with image)"
         case .gdsModalInfoView:
             return "Modal Info View"
+        case .gdsListOptions:
+            return "List Options"
         }
     }
     
     var isModal: Bool {
         switch self {
-        case .gdsInstructions, .gdsInstructionsWithImage:
```

Comment:
Do we want to remove these 2 if we are going down the route of having a default value instead? I had wanted to list them to ensure anyone adding a future one knew specifically how they were presenting it, but appreciate that maybe doesn't matter too much.

Reply:
I figured the important thing is to explicitly capture the unusual cases (ie modal) on the basis that almost all screens won't be presented modally. It seemed like that modal case was already going to get quite long with just 3 screens there which is why I adjusted it to have a default condition.

I don't have a strong feeling either way, just concerned about it becoming a bit unwieldy as we add several more screen patterns.

Reply:
I think given the list of views we'll be demoing here it would make sense to remove and add `default:` as the vast majority won't be modal. Equally don't have a strong feeling either way but yeah might get to be quite a long list in no time 🙂